templates/go/client.go.twig

```diff
@@ -65,6 +65,7 @@ func NewClient() Client {
 	headers := map[string]string{
 {% for key,header in spec.global.defaultHeaders %}
 		"{{key}}" : "{{header}}",
+		"user-agent" : "{{spec.title | caseUcfirst}}{{ language.name | caseUcfirst }}SDK/{{ sdk.version }}",
```

Comment:
In go, you can use `runtime.GOOS`:

```go
package main

import (
	"fmt"
	"runtime"
)

func main() {
	s := fmt.Sprintf("(%s;%s)", runtime.GOOS, runtime.GOARCH)
	fmt.Println(s)
}
```

outputs: 

```
(linux;amd64)
```

https://go.dev/play/p/aS0Ik4UnKXj

Reply:
updated